### PR TITLE
Add auto_punc and update docs.

### PR DESCRIPTION
**Pull request by KPatr1ck**
1. Add auto_punc model in  text/punctuation_restoration
2. Update docs of audio/tts modules
   - deepvoice3_ljspeech 
   - fastspeech_ljspeech
   - transformer_tts_ljspeech
   - lstm_tacotron2

### CODE REVIEW

File: modules/text/punctuation_restoration/ernie_punc/module.py [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+from paddlehub.utils.log import logger
+from paddlenlp.transformers import ErnieTokenizer, ErnieForTokenClassification
+from paddlenlp.data import Pad
+
+
+@moduleinfo(
+    name="ernie_punc",
+    version="1.0.0",
+    summary="",
+    author="Baidu",
```
Comment (nepeplwu) [inline modules/text/punctuation_restoration/ernie_punc/module.py]: 建议改为PaddlePaddle

File: modules/text/punctuation_restoration/ernie_punc/README.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
+# ernie_punc
+
+|模型名称|ernie_punc|
```
Comment (nepeplwu) [inline modules/text/punctuation_restoration/ernie_punc/README.md]: 建议去掉ernie，并且起个更表意的名字

File: modules/text/punctuation_restoration/ernie_punc/README.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+| :--- | :---: |
+|类别|文本-标点恢复|
```
Comment (nepeplwu) [inline modules/text/punctuation_restoration/ernie_punc/README.md]: 是否有更好的类别

File: modules/text/punctuation_restoration/ernie_punc/README.md (hunk 2 above)
Comment (KPatr1ck) [reply modules/text/punctuation_restoration/ernie_punc/README.md]: 改为auto_punc

File: modules/text/punctuation_restoration/ernie_punc/module.py (hunk 1 above)
Comment (KPatr1ck) [reply modules/text/punctuation_restoration/ernie_punc/module.py]: done

**Review by haoyuying — APPROVED** [review]
LGTM

**Pull request merged**